src/go/configgenerator/route_generator.go

```diff
@@ -38,7 +38,7 @@ const (
 	virtualHostName = "backend"
 )
 
-func MakeRouteConfig(serviceInfo *configinfo.ServiceInfo) (*routepb.RouteConfiguration, error) {
+func makeRouteConfig(serviceInfo *configinfo.ServiceInfo) (*routepb.RouteConfiguration, error) {
```

Comment:
Is changing to private really needed? Are you sure you will never call this directly?

Reply:
Good question. I think changing this to private is only for best practice. Functionality wise it's same. IMO it's better to keep these private until we really need it. Like MakeRouteTable we just made it public. I was curious why this name is public and the rest are private but in fact this is never used outside of the package :)